### PR TITLE
Fixing sprite player

### PR DESCRIPTION
**Pull request by chris-addison**
This fixes the issues with creating the sprite character. The look of startup still needs some work

### CODE REVIEW

File: Assets/tiny_rogue/Scripts/CreatureLibrary.cs [hunk 1]
```diff
@@ -223,7 +227,7 @@ public void FixupSpritePlayer(EntityCommandBuffer cb, Entity player)
             
             cb.AddComponent(player, att);
             cb.AddComponent(player, c);
-            cb.AddComponent(player, sight);
+            //cb.AddComponent(player, sight);
```
Comment (mdiluz) [inline Assets/tiny_rogue/Scripts/CreatureLibrary.cs]: Is this intended?

File: Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs [hunk 2]
```diff
@@ -291,17 +291,11 @@ protected override void OnUpdate()
...
-                                _creatureLibrary.SpawnPlayer(EntityManager);
-                            }
-                            else
-                            {
-                                Entities.WithAll<Player>().ForEach((Entity player) =>
-                                {
-                                    _creatureLibrary.FixupSpritePlayer(PostUpdateCommands, player);
-                                });
-                            }
+                                parent.Value = playerEntity;
```
Comment (unity-cwells) [inline Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs]: Is this going to break ascii?

File: Assets/tiny_rogue/Scripts/CreatureLibrary.cs (hunk 1 above)
Comment (unity-cwells) [inline Assets/tiny_rogue/Scripts/CreatureLibrary.cs]: The whole FixupSpritePlayer can be removed

**Review by unity-cwells — APPROVED** [review]

File: Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs (hunk 2 above)
Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs]: I'll go check!

File: Assets/tiny_rogue/Scripts/CreatureLibrary.cs (hunk 1 above)
Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/CreatureLibrary.cs]: Will do!

Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/CreatureLibrary.cs]: This method is unused now. Deleting it

File: Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs (hunk 2 above)
Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs]: Fixed

**Pull request merged**